src/richie/apps/search/templates/search/search.html

```diff
@@ -3,6 +3,7 @@
 {% get_current_language as LANGUAGE_CODE %}
 
 {% block meta %}
+  {{ block.super }}
```

Comment:
Nice :pray: 
Can you add an entry to the CHANGELOG.md file? And if you're in the mood maybe a small test in https://github.com/openfun/richie/blob/master/tests/apps/search/test_templates_search.py